dispatch/BESInterface.cc

```diff
@@ -411,9 +405,13 @@ int BESInterface::execute_request(const string &from)
 
     // TODO If this is only used for logging, it is not needed since the log has a copy
     //  of the BES PID. jhrg 11/13/17
+#if 0
```

Comment:
Should it stay or should it go...  (The Clash)

Reply:
It should go, but the things in data[] are fragile, so I replaced the ostringstream version with something that is faster.